src/Client/Web/Bit.Client.Web.BlazorUI/Components/SearchBoxes/BitSearchBox.razor.cs

```diff
@@ -63,6 +63,9 @@ public partial class BitSearchBox
                 _ = ValueChanged.InvokeAsync(value);
             }
         }
+        /// <summary>
+        /// Callback called when value is changed.
+        /// </summary>
         [Parameter] public EventCallback<string?> ValueChanged { get; set; }
```

Comment:
again, please remove this documentation comment.
it is not correct for these specific parameters.